src/webview/css/css.js

```diff
@@ -108,6 +108,16 @@ hr {
 .message-brief {
   padding: 0 1em 1em 4em;
 }
+.message-brief > .content > :first-child {
+  clear: right;
+}
+.message-brief > .content > :last-child::after {
```

Comment:
Are you sure you need the `:after' pseudo-element? Can you achieve this in a more conventional way?
You should definitely use the already existing `.timestamp' styling, just adding the differentce in styles for `.message-brief `. That will ensure the styles change together.

Reply:
Implementing this conventionally would be more complicated and would require an `::after` pseudo element in any case in order to give a sort of `left-margin` to the message because of the reasons mentioned in #3375 
>One part of how to pull that off is that it's important the actual message text not overlap the timestamp, or even crowd it too closely

>will ensure the styles change together.

Makes sense, thanks!